drivers/led_strip/lpd880x.c

```diff
@@ -78,7 +78,7 @@ static int lpd880x_update(const struct device *dev, void *data, size_t size)
 
 	rc = spi_write_dt(&config->bus, &tx);
 	if (rc) {
-		LOG_ERR("can't update strip: %d", rc);
+		LOG_ERR("can't update strip: %lu", rc);
```

Comment:
Hi @mateusz-holenko,

Thanks for this patch. Are you compiling for a 64-bit target ?

`size_t` is `unsigned int` for 32-bit but `long unsigned int` for 64-bit. So I suggest we use `%zu` here? It the length specifier dedicated to `size_t` (C99).

Reply:
Hi @simonguinot, thanks for the suggestion. It's fixed now.

For the record I based my previous solution on an error observed in Zephyr CI (https://github.com/zephyrproject-rtos/zephyr/actions/runs/10997274239/job/30536320469#step:12:332) which claimed:
```/__w/zephyr/zephyr/drivers/led_strip/lpd880x.c:81:25: error: format '%d' expects argument of type 'int', but argument 2 has type 'size_t' {aka 'long unsigned int'} [-Werror=format=]```
It was targetting `intel_adl_crb` (`alder_lake`) so yes, a 64-bit target.